adapters/33across/33across/exemplary/simple-video.json

```diff
@@ -0,0 +1,110 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "video": {
+          "w": 728, 
+          "h": 90,
+          "protocols": [2],
+          "placement": 1,
+          "startdelay": -2,
+          "playbackmethod": [2],
+          "mimes": ["foo", "bar"]
+        },
+        "ext": {
+          "bidder": {
+            "siteId": "fake-site-id",
+            "productId": "instream"
+          }
+        }
+      }
+    ],
+    "site": {}
+  },
+
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://ssc.33across.com",
+        "body": {
+          "id": "test-request-id",
+          "imp": [
+            {
+              "id":"test-imp-id",
+              "video": {
+                "w": 728, 
+                "h": 90,
+                "protocols": [2],
+                "placement": 1,
+                "startdelay": -2,
+                "playbackmethod": [2],
+                "mimes": ["foo", "bar"]
+              },
+              "ext": {
+                "ttx": {
+                  "prod": "instream"
+                }
+              }
+            }
+          ],
+          "site": {
+            "id": "fake-site-id"
+          }
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "id": "test-request-id",
+          "seatbid": [
+            {
+              "seat": "ttx",
+              "bid": [{
+                "id": "8ee514f1-b2b8-4abb-89fd-084437d1e800",
+                "impid": "test-imp-id",
+                "price": 0.500000,
+                "adm": "some-test-vast-ad",
+                "crid": "crid_10",
+                "h": 90,
+                "w": 728,
+                "ext": {
+                  "ttx": {
+                    "mediaType": "video"
+                  }
```

Comment:
Do we only expect the 33Across server to respond with an `"ext"` field to video requests? Because in the banner example found in `adapters/33across/33across/exemplary/simple-banner.json` the `"mockResponse"` doesn't have that. If they do, should we expect a `"banner"` value?
```
45       "mockResponse": {
46         "status": 200,
47         "body": {
48           "id": "test-request-id",
49           "seatbid": [
50             {
51               "seat": "ttx",
52               "bid": [{
53                 "id": "8ee514f1-b2b8-4abb-89fd-084437d1e800",
54                 "impid": "test-imp-id",
55                 "price": 0.500000,
56                 "adm": "some-test-ad",
57                 "crid": "crid_10",
58                 "h": 90,
59                 "w": 728
   +               "ext": {
   +                 "ttx": {
   +                   "mediaType": "video"
   +                 }
   +               }
60               }]
61             }
62           ],
63           "cur": "USD"
64         }
65       }
66     }
67   ],
adapters/33across/33across/exemplary/simple-banner.json
```

Reply:
Correct, the `ext` field if present will specify the mediaType. If not present assume banner